tests/unit/multi-currency/test-class-multi-currency.php

```diff
@@ -21,19 +21,28 @@ class WCPay_Multi_Currency_Tests extends WP_UnitTestCase {
 	public function setUp() {
 		parent::setUp();
 
+		$this->mock_currency_settings(
+			'GBP',
+			[
+				'price_charm'    => '-0.1',
```

Comment:
Only issue I have with this is that it's -0.1 instead of -0.01, which doesn't matter because the calculation and not the amount is what matters.

Reply:
I thought -0.10 would be a valid charm value when writing these tests initially. But you're right, we're testing the calculations and not the amount here. I think we could keep it as is to avoid having to update all tests as well.